homeassistant/components/airzone/select.py

```diff
@@ -0,0 +1,158 @@
+"""Support for the Airzone sensors."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any, Final
+
+from aioairzone.common import GrilleAngle, SleepTimeout
+from aioairzone.const import (
+    API_COLD_ANGLE,
+    API_HEAT_ANGLE,
+    API_SLEEP,
+    AZD_COLD_ANGLE,
+    AZD_HEAT_ANGLE,
+    AZD_NAME,
+    AZD_SLEEP,
+    AZD_ZONES,
+)
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import AirzoneUpdateCoordinator
+from .entity import AirzoneEntity, AirzoneZoneEntity
+
+
+@dataclass
+class AirzoneSelectDescriptionMixin:
+    """Define an entity description mixin for select entities."""
+
+    api_param: str
+    options: dict[str, int]
```

Comment:
Options is now provided by the base select but it needs to be a list

https://github.com/home-assistant/core/blob/3dba9c469584223cd693d01b3231453d2ab422da/homeassistant/components/select/__init__.py#L102

Reply:
@bdraco should be addressed in 7b112fe178929f5057e7ed4392d0a8c0ecfbaf31